1_For_Loops.ipynb

```diff
@@ -202,7 +202,7 @@
   },
```

Comment:
<p>good job! </p><p>try to solve this using unpacking :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week04-beckyrosenfeld/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='111'/>